src/utils/ArgsOptions.cpp

```diff
@@ -56,6 +57,26 @@ int ArgsOptions::get_cells_size()
     return _k_cells_size;
 }
 
+void ArgsOptions::set_cell_bitsize(int cell_bitsize)
+{
+    switch (cell_bitsize) {
```

Comment:
I'm sure that this can be simplified to a simple `if` statement like:

```
if (cell_bitsize != 32) {
    _k_cell_bitsize = cell_bitsize;
} else {
    _k_cell_bitsize = 32;
}
```

Reply:
Hi @rafaelcn, actually I am taking care of all cases that are different from `8`, `16`, 32`, `64`. For example, if the user passes `brain blah.b --size-cell=20`, that is not a valid code. In my version, that would fall in the `default` clause generating later the LLVM type `Int32Ty`, Brain's default value. In yours, that would set the cell bitsize to `20`, making it incorrect, once LLVM would fail in generating the type `Int20Ty`. :)

Reply:
You’re right, I didn’t think of that.